operate/schema/pom.xml

```diff
@@ -38,18 +38,143 @@
       </exclusion>
       </exclusions>
     </dependency>
-	</dependencies>
 
-	<build>
-		<plugins>
+    <dependency>
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-lang3</artifactId>
+      <scope>test</scope>
+    </dependency>
+
+  </dependencies>
+
+  <build>
+    <plugins>
+
+      <!-- LICENSE PLUGIN -->
+      <plugin>
+        <groupId>com.mycila</groupId>
+        <artifactId>license-maven-plugin</artifactId>
+      </plugin>
+      
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-failsafe-plugin</artifactId>
+      </plugin>
+
+    </plugins>
+ 
+  </build>
+
+   
+  <profiles>
+    <profile>
+      <id>docker</id>
+      <activation>
+        <property>
+          <name>!skipDockerProfile</name>
+        </property>
+      </activation>
+      <build>
+        <plugins>
+
+          <plugin>
+            <groupId>org.codehaus.mojo</groupId>
+            <artifactId>exec-maven-plugin</artifactId>
+            <executions>
+              <execution>
+                <id>docker-start</id>
+                <phase>pre-integration-test</phase>
+                <goals>
+                  <goal>exec</goal>
+                </goals>
+                <configuration>
+                  <skip>${skipTests}</skip>
+                  <executable>docker-compose</executable>
+                  <workingDirectory>..</workingDirectory>
+                  <arguments>
+                    <argument>up</argument>
+                    <argument>-d</argument>
+                    <argument>elasticsearch</argument>
+                  </arguments>
+                </configuration>
+              </execution>
+
+              <execution>
+                <id>docker-stop</id>
+                <phase>post-integration-test</phase>
+                <goals>
+                  <goal>exec</goal>
+                </goals>
+                <configuration>
+                  <skip>${skipTests}</skip>
+                  <executable>docker-compose</executable>
+                  <workingDirectory>..</workingDirectory>
+                  <arguments>
+                    <argument>down</argument>
+                  </arguments>
+                </configuration>
+              </execution>
+            </executions>
+          </plugin>
+        </plugins>
+      </build>
+    </profile>
+    <profile>
+      <id>docker-opensearch</id>
```

Comment:
I think we should keep changes related to running our ITs under opensearch as a separate effort. There is other work that needs to be done as part of this, as well as the IT refactoring effort, and it would be good to keep changes matched with individual tasks and epics

Reply:
Fair point. Just to make sure we mean the same thing: I understand it that we still make this particular change now, just have separate commits/PRs and link it with a different ticket, right?

Even if it is only a small part of the Opensearch testing story, this profile helped us already quite a bit with developing as we found a bug in the Opensearch code that we would have otherwise found only later (e.g. after release like the one Ralf found recently).

Reply:
That makes sense. If it's useful as part of the testing, that's fine. I was more concerned we were also pulling in the whole task for re-running all the tests with Opensearch as part of the pipeline and wanted to keep that separate since this is already a large effort. If this profile though is useful for testing and for this task, it makes perfect sense thank you.